labsdk/_test/main.py

```diff
@@ -20,15 +20,15 @@
 # getting started code
 
 @raptor.register(int, staleness='10h')
-@raptor.connector("emails")  # <-- we are decorating our feature with our production data-connector! 😎
+@raptor.source("emails")  # <-- we are decorating our feature with our production DataSource! 😎
```

Comment:
`data_source`

I think there are a few times when you short hand to `source` - I might suggest not doing this for clarity reasons.  Just use Data Source?